gigascience/gigadb-website
# Able to toggle expand dataset sample attributes field

# Pull request for issue: #1849

This is a pull request for the following functionalities:

* Makes "+" butotn toggleable in e.g. http://gigadb.gigasciencejournal.com/dataset/100006

![image](https://github.com/gigascience/gigadb-website/assets/143437854/7efd2225-c5a8-4ca8-bc70-67f895139e89)

![image](https://github.com/gigascience/gigadb-website/assets/143437854/57bb518c-8a0e-4026-9c76-18c47a18a8a8)

## How to test?

Go to above page and confirm:
* clicking on + button expands content and shows a "-" button
* clicking on the "-" button collapses content
* + button has an aria-expanded="false" attribute and an aria-label="show more"
* - button has analogous attributes

## How have functionalities been implemented?

* Updated js code to toggle the button isntead of hiding it
* created new button style to prevent button content from being hidden on the focused state

## Any issues with implementation?

--

## Any changes to automated tests?

--
